twigs/ecr.py

```diff
@@ -0,0 +1,172 @@
+import sys
+import os
+import subprocess
+import logging
+import json
+
+from . import docker
+
+
+g_encoding = None
+
+def set_encoding(encoding):
+    global g_encoding
+    g_encoding = encoding
+
+def get_encoding():
+    global g_encoding
+    return g_encoding
+
+def get_repo_uri(repository, repositoryType):
+
+    if repositoryType == 'private':
+        repo_cmd = "ecr describe-repositories"
+    else:
+        repo_cmd = "ecr-public describe-repositories"
+
+    t_json = run_aws_cmd(repo_cmd)
```

Comment:
Just wanted to check if there is any additional dependency or existing AWS CLI (which is also required by prowler for AWS_CIS benchmarks) suffices?

Reply:
No there is no additional dependency.